src/main/java/uk/gov/hmcts/reform/propertiesvolume/PropertiesvolumePropertySourceLocator.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.propertiesvolume;
+
+import org.springframework.cloud.bootstrap.config.PropertySourceLocator;
+
+import org.springframework.core.annotation.Order;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.Environment;
+import org.springframework.core.env.MapPropertySource;
+
+/**
+ * Properties volume {@link PropertySourceLocator} (e.g. keyvault flexvolume mount).
+ *
+ */
+@Order(0)
+public class PropertiesvolumePropertySourceLocator implements PropertySourceLocator {
+
+    private final transient PropertiesvolumeConfigProperties properties;
```

Comment:
didn't seem like the right order to me but looks good 👍 
https://stackoverflow.com/a/16732059